reference/commands/graph/info_format_json.rst

```diff
@@ -0,0 +1,229 @@
+.. _reference_commands_graph_info_json_format:
```

Comment:
Problem is that files need to be referenced in the TOC, not only cross-referenced

Alternatives are:
Reference->Commands->New "output" section
Reference->Extensions->API->New "output" section

Where "output" can be "Formatters output", our "Output files format", or something like that, that explains the graph.html, the graph.json, the ``pkg-list.json`` once it is released, etc.